docs/guides/expectations/features_custom_expectations/how_to_add_input_validation_for_an_expectation.md

```diff
@@ -1,137 +1,115 @@
 ---
-title: How to add configuration validation for an Expectation. 
+title: How to add Input Validation and Type Checking for a Custom Expectation 
 ---
 
+import Prerequisites from '../creating_custom_expectations/components/prerequisites.jsx'
+
 This document describes how to add configuration validation for an Expectation. 
 
-:::note Prerequisites
-This how-to guide assumes you have already:
+<Prerequisites>
 
-* [Set up a working deployment of Great Expectations](/docs/tutorials/getting_started/intro)
-:::
+ - Created a [Custom Expectation](../creating_custom_expectations/overview.md)
+
+</Prerequisites>
+
+[Expectations](../../../reference/expectations/expectations.md) will typically be configured using input parameters: 
+for example, we might expect the fraction of null values to be most `mostly=.05`, or the mean of a row to adhere to a `min_value=5` bound.
```

Comment:
If you remove line 7 and have the introduction to this guide start here, it would make sense to tweak the initial statement to reference type checking and validation of input parameters.  (It makes less sense to have a guide about type checking and validation of input parameters start out by not mentioning them and talking about Expectations instead.)

```suggestion
[Expectations](../../../reference/expectations/expectations.md) will typically be configured using input parameters which the Expectation's code will require to fulfill some criteria in order to be used or provide relevant results.  Ensuring that these requirements are fulfilled is the purpose of Type Checking and validating your input parameters.

For example, we might expect the fraction of null values to be most `mostly=.05`, in which case any value above 1 would indicate an impossible fraction of a single whole (since a value above one indicates more than a single whole), and should throw an error.  Another example would be if we want to indicate that the the mean of a row adheres to a minimum value bound, such as `min_value=5`.  In this case, attempting to pass in a non numerical value should clearly throw an error!
```